pkg/config/config.go

```diff
@@ -378,15 +388,15 @@ func loadProxyFromEnv() {
 		}
 	}
 
-	if HTTP := getEnvCaseInsensitive("HTTP_PROXY"); HTTP != "" {
+	if HTTP := getEnvDDPrecedence("HTTP_PROXY"); HTTP != "" {
```

Comment:
I would indeed be in favor of `DD_PROXY_HTTP` to be consistent with the rest of the configuration. `HTTP_PROXY` is an exception because it's kind of a standard on linux.